### PR TITLE
Revise narrow container width rules and apply to all view headers.

### PR DESCRIPTION
**Pull request by tfrcarvalho**
* Apply narrow containers to view headers to keep sync with the content.
* Fix a lack of sync bug between headers and contents in narrow windows.

# Type of change

Bug fix and clean up.

# Test and Reproduce

Check all pages that use NarrowContainer to check for components with different width behaviors.

Screen Shot 2019-04-05 at 5 49 17 PM

### CODE REVIEW

File: app/assets/src/components/layout/narrow_container.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .narrowContainer {
-  max-width: 1280px;
+  max-width: 1308px;
```
Comment (gregdingle) [inline app/assets/src/components/layout/narrow_container.scss]: please add a comment to explain how this value was chosen

File: app/assets/src/components/layout/narrow_container.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   margin: 0 auto;
-  width: 100%;
+  width: calc(100% - 28px);
```
Comment (gregdingle) [inline app/assets/src/components/layout/narrow_container.scss]: please add a comment to explain the `-28px`

File: app/assets/src/components/layout/narrow_container.scss (hunk 1 above)
Comment (tfrcarvalho) [reply app/assets/src/components/layout/narrow_container.scss]: done

File: app/assets/src/components/layout/narrow_container.scss (hunk 2 above)
Comment (tfrcarvalho) [reply app/assets/src/components/layout/narrow_container.scss]: done

**Pull request merged**